openedx/core/lib/tests/test_studio_entities.py

```diff
@@ -0,0 +1,14 @@
+"""Tests for the Studio entity plugin API."""
+from django.test import TestCase
+
+from openedx.core.lib.api.plugins import PluginError
+from openedx.core.lib.studio_entities import StudioEntityPluginManager
+
+
+class TestStudioEntityPluginApi(TestCase):
+    """Unit tests for the Studio entity plugin API."""
+
+    def test_get_invalid_plugin(self):
```

Comment:
Can you add a test for ``get_enabled_entities`` to verify the behavior with entities that are both enabled and disabled.